beta/src/content/learn/keeping-components-pure.md

```diff
@@ -688,7 +688,7 @@ export default function StoryTray({ stories }) {
           {story.label}
         </li>
       ))}
-      <li>Create Story</li>
+      <li key={'create'}>Create Story</li>
```

Comment:
We will need key only when we use map or other loop to render elements dynamically. I see that this `li` is outside the map so we won't need a key here.

Reply:
that’s correct

Reply:
I will close the PR then, thanx.